.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
```

Comment:
Remove this file from your PR.
PS : this is a tough job, first try doing it yourself, otherwise ask me on zulip/slack.